containers.py

```diff
@@ -0,0 +1,23 @@
+from dependency_injector import providers, containers
+
+# services
+from services.language_management import LanguageManagement
+
+# gtk
+from gtk_gui.custom_modal import CustomModalGtk
+from gtk_gui.welcome_window import WelcomeWindowGtk
+
+
+class Configs(containers.DeclarativeContainer):
+    config = providers.Configuration('config')
+    config_main_window_page = providers.Configuration('config')
+
+
+class Locales(containers.DeclarativeContainer):
+    locale_general = providers.Singleton(LanguageManagement, None, None, None)
+
+
+class GraphicGui(containers.DeclarativeContainer):
```

Comment:
That graphic-thingy is in a common file, which creates an anomaly as you wan't common files between different front-ends (gtk, ncurses etc.) to not depend on a specific frontend.